Update term for STS releases (#48127)
dotnet/docs: docs/core/releases-and-support.md

* *Standard Term Support* (STS) releases

>   These versions are supported for 2 years (24 months).

Reviewer: ```suggestion
  These versions are supported for two years (24 months).
```

  Example:

>   * .NET 9 is an STS release and was released in November 2024. It's supported for 2 years, until November 2026.

Reviewer: ```suggestion
  * .NET 9 is an STS release and was released in November 2024. It's supported for two years, until November 2026.
```

* *Long Term Support* (LTS) releases

  These versions are supported for a minimum of 3 years, or 1 year after the next LTS release ships if that date is later.

  Examples:

>   * .NET Core 3.1 is an LTS release and was released in December 2019. It was supported for 3 years, until December 2022.

Reviewer: ```suggestion
  * .NET Core 3.1 is an LTS release and was released in December 2019. It was supported for three years, until December 2022.
```

>   * .NET 8 is an LTS release and was released in November, 2023. It's supported for 3 years, until November 2026.

Reviewer: ```suggestion
  * .NET 8 is an LTS release and was released in November 2023. It's supported for three years, until November 2026.
```

Releases alternate between LTS and STS.

Servicing updates ship monthly and include both security and non-security (reliability, compatibility, and stability) fixes. Servicing updates are supported until the next servicing update is released. Servicing updates have runtime roll forward behavior. That means that applications default to running on the latest installed runtime servicing update.

## How to choose a release

> If you're building a service and expect to continue updating it on a regular basis, then an STS release like the .NET 7 runtime might be your best option to stay up to date with the latest features .NET has to offer.

Reviewer: ```suggestion
If you're building a service and expect to continue updating it on a regular basis, then an STS release like the .NET 9 runtime might be your best option to stay up to date with the latest features .NET has to offer.
```

Reviewer: this paragraph is a bit misleading though. you should always pick the latest (STS or LTS) if you're updating on a regular basis

> If you're building a client application that will be distributed to consumers, stability might be more important than access to the latest features. Your application might need to be supported for a certain period before the consumer can upgrade to the next version of the application. In that case, an LTS release like the .NET 6 runtime could be the right option.

Reviewer: ```suggestion
If you're building a client application that will be distributed to consumers, stability might be more important than access to the latest features. Your application might need to be supported for a certain period before the consumer can upgrade to the next version of the application. In that case, an LTS release like the .NET 8 runtime could be the right option.
```

* *Standard Term Support* (STS) releases

  These versions are supported for two years (24 months).

  Example:

  * .NET 9 is an STS release and was released in November 2024. It's supported for two years, until November 2026.

* *Long Term Support* (LTS) releases

  These versions are supported for a minimum of 3 years, or 1 year after the next LTS release ships if that date is later.

>   Examples:

Author: ```suggestion
  Example:
```

>   * .NET Core 3.1 is an LTS release and was released in December 2019. It was supported for three years, until December 2022.

Reviewer 2: I would remove the .NET Core 3.1 example, it is too far back.

>   * .NET Core 3.1 is an LTS release and was released in December 2019. It was supported for three years, until December 2022.

Author: ```suggestion
```

## Supported operating systems

.NET can be run on a range of operating systems. Each of these operating systems has a lifecycle defined by its sponsor organization (for example, Microsoft, Red Hat, or Apple). These lifecycle schedules are taken into account when .NET adds and removes support for operating system versions.

> When an operating system version goes out of support, Microsoft stops testing and providing support for that version. Users need to move forward to a supported operating system version to get support.

Reviewer 2: ```suggestion
When an operating system version reaches end of support, Microsoft stops testing and providing support for that version. Users need to move forward to a supported operating system version to get support.
```
